README.md

```diff
@@ -43,6 +43,7 @@ You can use environment variables in order to set up all options. You can set de
 - `BUGSNAG_USE_LOGGER`
 - `BUGSNAG_RELEASE_STAGE`
 - `BUGSNAG_NOTIFY_RELEASE_STAGES`
+- `HOSTNAME`
```

Comment:
This should be updated to correspond to the new default.

Reply:
My mistake, totally forgot to update in both spots